### DISCUSSION TITLE
Example of using CodeCeptJs with AWS Device Farm

### DISCUSSION THREAD

**Discussion by bou2you** [codeceptjs/CodeceptJS#3070] [Q&A]
I love CodeCeptJs and would like to use it with AWS Device Farm, for both Appium-based mobile web & native app testing as well as WebDriver desktop browser testing (it seems AWS Device Farm distinguishes these).

I just wanted to ask for help here before embarking on trying to get it to work myself.

Has anyone come across any working sample code online I could follow?

I tried googling but didn't find concrete examples for this technology stack - just that it should be possible.

I will have to follow AWS documentation, then CodeCeptJs documentation, and then try to connect the two I guess..

**Comment by ukiras123** [comment]
Were you able to figure it out? @bou2you

**Reply by bou2you** [reply]
No.

I just went with the already documented [CodeCeptJs + TestingBot integration](https://testingbot.com/support/getting-started/codeceptjs.html) and moved on.
